### ISSUE TITLE
Trackpad tap to click?

### ISSUE THREAD

**Issue by MansonEsqueEgo** [Apricity-OS/apricity-build#116]
I just installed the 07/2016 build (cinnamon), and tap to click isn't working, i did enable it in settings, but perhaps i'm missing something obvious, if so my apologies. Thank you for the amazing work!

**Comment by dealloc** [comment]
There's been more issues like this, it's been answered in #93

**Comment by MansonEsqueEgo** [comment]
I looked at that first, the thing is unless i messed up in the process, ive already followed that guide, ( uninstalling what was mentioned) and after restarting, i still get that bug

**Comment by dealloc** [comment]
Can you try my suggestion of running

 `synclient TapButton1=1 TapButton2=3 TapButton3=2`

in your terminal and verify again?

**Comment by MansonEsqueEgo** [comment]
Thank you so much! Though when my nooby self tries to run it all that comes back is "synclient: command not found"

**Comment by dealloc** [comment]
My first guess would be to try and install the synclient package :)

**Comment by MansonEsqueEgo** [comment]
Thank you so very much! That did it, at last! I had that package installed, but for some reason whenever i ran what you sent me, the output (yeah noob language for the win) stated i didn't have the proper drivers, so i reinstalled the package, rebooted and actually took the time to browse through the wiki this time. Sorry about my utter ignorance, but I'm just getting started on linux, I've pretty much just distro hopped thus far...

**Comment by RubenKelevra** [comment]
Seems a bit off-question but on-topic: I think the tap to click option should be enabled after installation. This is the default option which seems to be expected by the most users.

**Comment by dealloc** [comment]
I've seen it disabled out of the box on pretty much any distro I've used with Gnome

**Comment by RubenKelevra** [comment]
Right, a friend created a bugreport on gnome about that, because the usual expectation from $users are more like this is working. But they decided to let it disabled.

Surely this might be a personal choice, but I think the default should be enabled, because everybody expect that.

**Comment by bitnukl** [comment]
I would also vote for enabled tap to click by default. It's what users from Windows and OSX expect and it makes sense.
I also can't see any downside, accidental clicks should be extremely rare. Users bothered by that can deactivate it anytime, disabling touchpad while typing could also help.

**Comment by RubenKelevra** [comment]
Thanks @bitnukl that is my point.

**Comment by dealloc** [comment]
Main painpoint for me would be accidently touching the touchpad while typing, especially with my touchpad being placed rather awkardly

**Comment by RubenKelevra** [comment]
Yeah, some devices got this issue, but still the sane default should be, what all users expect not the edge-cases or workarounds of hardware-issues.
